### DISCUSSION TITLE
Does db-scheduler supports MySql Cluster  mode?

### DISCUSSION THREAD

**Discussion by tianshuainan** [kagkarlsson/db-scheduler#474] [Q&A]
now, I want to extend mysql single-node to master-slave mode.Does db-scheduler supports MySql Cluster  mode?

**Comment by kagkarlsson** [comment]
Not sure what master-slave mode is. If it is simple replication and you still have a master it should work. If it is some multi-master setup using Galera or similar, not so sure. I think someone tried that but observed some errors.
